zingocli/src/lib.rs

```diff
@@ -13,70 +13,59 @@ use zingolib::{commands, lightclient::LightClient};
 
 pub mod version;
 
-pub fn build_clap_app() -> clap::App<'static> {
-    clap::App::new("Zingo CLI").version(version::VERSION)
+pub fn build_clap_app() -> clap::ArgMatches {
+    clap::Command::new("Zingo CLI").version(version::VERSION)
             .arg(Arg::new("nosync")
                 .help("By default, zingo-cli will sync the wallet at startup. Pass --nosync to prevent the automatic sync at startup.")
                 .long("nosync")
-                .short('n')
-                .takes_value(false))
```

Comment:
i see you have removed all the `.takes_value`s, does this mean that whether a value is required with the flag is no  longer enforced?

Reply:
oh, but there are instances where both take_values(true) and take_values(false) have been removed